packages/aws-cdk/lib/commands/context.ts

```diff
@@ -88,7 +88,7 @@ function invalidateContext(context: Context, key: string) {
   // Unset!
   if (context.has(key)) {
     context.unset(key);
-    print(`Context value ${colors.blue(key)} reset. It will be refreshed on the next SDK synthesis run.`);
+    print(`Context value ${colors.blue(key)} reset. It will be refreshed on next synthesis`);
```

Comment:
Maybe this should hint that it means network + credentials are needed on the next execution.